lib/grammers-mtproto/src/mtp/encrypted.rs

```diff
@@ -114,9 +116,15 @@ impl Builder {
         self
     }
 
-    /// Configures the compression threshold for outgoing messages.
-    pub fn compression_threshold(mut self, threshold: Option<usize>) -> Self {
-        self.compression_threshold = threshold;
+    /// Configures the min compression threshold for outgoing messages.
```

Comment:
```suggestion
    /// Configures the lower bound of the compression threshold for outgoing messages.
```

Perhaps we should remove the `Option`? A value of `0` should do the same.